verify/check.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/vonage/vonage-go-sdk"
+)
+
+func main() {
+	godotenv.Load("../.env")
+
+	auth := vonage.CreateAuthFromKeySecret(os.Getenv("VONAGE_API_KEY"), os.Getenv("VONAGE_API_SECRET"))
+	verifyClient := vonage.NewVerifyClient(auth)
+
+	response, errResp, err := verifyClient.Check(os.Getenv("REQUEST_ID"), os.Getenv("CODE"))
+
+	if err != nil {
+		fmt.Printf("%#v\n", err)
+	} else if response.Status != "0" {
+		fmt.Println("Error status " + errResp.Status + ": " + errResp.ErrorText)
+	} else {
+		// all good
```

Comment:
This comment feels like it sticks out a bit. I would guess it was there when writing the code?

Reply:
Yes I think if we don't have other comments, this shouldn't be here either. Removed.